PeerCo/small_scale_self_consumed_solar.md

```diff
@@ -0,0 +1,289 @@
+
+## Applicability conditions
+
+This section lays out the requirements on a given context for this methodology to be applicable.
+
+### System eligibility requirements
+
+The system to which the methodology is applied shall meet the following requirements to be eligible:
+
+1. the rating of the PV is between 30 kWp and 5 MWp
+
+2. the system is a fixed installation, meaning that the PV system is mounted on a unique specific site where it will operate throughout the project
+
+3. generation is measured by the inverter(s) or a digital meter with high resolution data, which can either be communicated automatically by digital means (e.g. Rest API) or retrieved manually
+
+4. the asset owner documents the following:
+
+    a. PV asset type and brand
+
+    b. location of the installation (address, or GPS coordinates if an address is not available)
+
+    c. date of installation
+
+5. the power produced by the PV asset does not displace other on-site power generation (i.e. there is no on-site power generation whose output is affected by the generation of the PV asset)
+
+6. the asset owner or operator is not already being rewarded for self-consumption by another entity (e.g. government subsidies, ...)
+
+7. site demand (either as total demand or grid import) is measured by a digital meter with high resolution data, which can either be communicated automatically by digital means or retrieved manually
+
+In cases where power is exported from the site (i.e. not all solar generation is self-consumed), the following requirements shall also be met:
+
+8. the electricity export from the site is measured by a digital meter with high resolution data, which can either be communicated automatically by digital means or retrieved manually.
+
+### Requirements on the system during the project lifetime
+
+During the whole duration of the project, the PV asset owner shall demonstrate proof of ownership of the asset and continued operation of the self-consumption. This documentation shall be made via all of the following:
+
+- sharing of consumption data throughout the period,
+
+- in the case of usage of the environmental attribute in reporting:
+    - contractual renunciation to the possibility of transfer of the environmental assets to a third party.
+
+- in the case of sales of the environmental attribute to a third party:
+    - contractual transfer of the environmental attribute ownership of the self-consumed generation,
+    - carbon accounting for the site (location and market-based) should reflect the sale of the carbon reduction corresponding to the transfer of the certificate.
+
+
+### Requirements on the operating context
+
+The project country shall meet the following criteria:
+1. delivery of micro-solar is not a requirement for building development or to meet an enforced carbon quota,
+2. emission intensity data is available at the hourly (or sub-hourly) level for the national or local grid.
```

Comment:
intensity data at the national level does not align with my understanding of how OpenEAC was intended to function (maybe it's me that's wrong there!). If this would be sufficient in exceptional circumstances, we need to say clearly what those are

Reply:
As a specific example of country-wide data: France where regional data is not available today. Data granularity varies a great deal from one place to another, and we therefore need to support a variety of cases - with, of course, the principle that good local data is always better than national or worse.

More details on this in the next answer to your comment below